packages/react-components/src/components/Tag/Tag.tsx

```diff
@@ -118,6 +118,7 @@ export const Tag: React.FC<TagProps> = ({
           className={styles[`${baseClass}__remove`]}
         >
           <Icon
+            data-dismiss-icon
```

Comment:
Why it's needed?

Reply:
It's for preventing the list open in the case where a user wants to remove the selected item in multipicker mode.
[here](https://github.com/livechat/design-system/pull/357/files#diff-f098634dfe943b4c32066c5aeb6c944f3ffb3afd62308e62a43be6d6bcfd33b5R86)